.github/actions/php-pipeline/setup-env.sh

```diff
@@ -1,32 +1,28 @@
 #!/bin/bash
 
```

Comment:
This file could probably also use
```suggestion

set -eo pipefail

```

That'll also allow for getting rid of lines 41–45, since if the `git clone` fails it'll automatically exit.